### ISSUE TITLE
There is no documentation available

### ISSUE THREAD

**Issue by danryan** [revily/revily#14]
This is a serious problem that will prevent anyone from using reveille. Fix this before starting more shiny things. Looking at you, @danryan. _cough_

**Comment by trobrock** [comment]
@danryan agreed, tried to get it on heroku today but was thwarted but old docs. If you explain it to me enough to get going I can update the install docs.

**Comment by mxstrand** [comment]
Hi Dan - I too tried to fire up an instance of revily and ran into issues. Specifically, I:
-  Cloned the repo
-  $ bundle install

It got a few gems and then I received:

"Could not find rails_best_practices-1.15.0 in any of the sources"

So, I deleted the Gemfile.lock and then "$ bundle install" completes ok. Then I:
- Installed graphviz as per post-install message from ruby-graphviz

Next, as I try to run $ rake secret, I get the following error:

NoMethodError: undefined method "to_sym" for nil:NilClass
/Users/revily/config/application.rb:43:in "class:Application"

The offending line 43 specifically is:

`config.action_mailer.delivery_method = (ENV["MAILER_DELIVERY_METHOD"].to_sym || :smtp)`

Any ideas or guidance on this? I'm happy to do a PR into the docs too. Thanks sir.

**Comment by mpalmer** [comment]
You'd need to run `MAILER_DELIVERY_METHOD=smtp rake secret` to avoid that error.  That line of code is wrong, though, it should be

```
config.action_mailer.delivery_method = (ENV["MAILER_DELIVERY_METHOD"] || :smtp).to_sym
```
